mwdb/web/src/types/types.ts

```diff
@@ -0,0 +1,211 @@
+export type Capabality =
+    | "personalize"
+    | "share_queried_objects"
+    | "adding_comments"
+    | "manage_profile"
+    | "removing_parents"
+    | "adding_blobs"
+    | "removing_tags"
+    | "access_uploader_info"
+    | "removing_comments"
+    | "removing_objects"
+    | "adding_all_attributes"
+    | "unlimited_requests"
+    | "karton_assign"
+    | "karton_unassign"
+    | "reading_all_attributes"
+    | "adding_configs"
+    | "adding_files"
+    | "adding_tags"
+    | "sharing_with_all"
+    | "manage_users"
+    | "karton_reanalyze"
+    | "access_all_objects"
+    | "removing_attributes"
+    | "adding_parents";
+
+export type User = {
+    login: string;
+    groups: string[] | Group[];
+    capabilities: Capabality[];
+    additional_info?: string;
+    api_keys?: ApiKey[];
+    disabled?: boolean;
+    email?: string;
+    feed_quality?: string;
+    pending?: boolean;
+    requested_on?: string | Date;
+    provider?: null | string;
+    token?: string;
+    logged_on?: string | Date;
+    registered_on?: string | Date;
+    registrar_login?: string | Date;
+    set_password_on?: string | Date;
+};
+
+export type Group = {
+    admins: string[];
+    capabilities: Capabality[];
+    default: boolean;
+    immutable: boolean;
+    name: string;
+    private: boolean;
+    users: string[];
+    workspace: boolean;
+};
+
+export type ObjectType = "file" | "blob" | "config";
```

Comment:
API object type descriptors are a bit different:
```suggestion
export type ObjectType = "file" | "text_blob" | "static_config";
```